.devcontainer/Dockerfile

```diff
@@ -31,8 +31,9 @@ RUN apt-get install --assume-yes \
   "git=1:2.*" \
   "zsh=5.*"
 
-COPY "./scripts/*" "$HOME/"
-RUN curl -fsSL "https://github.com/cashapp/hermit/releases/download/v0.26.1/install.sh" | bash
+COPY ./scripts/ /root/
+
+RUN curl -fsSL "https://github.com/cashapp/hermit/releases/download/stable/install.sh" | bash
```

Comment:
Based on [this recent comment](https://github.com/cashapp/hermit/issues/345#issuecomment-1326834834), this line can be removed from `Dockerfile`. Hermit will only use `stable` channel in any case, as defined in the `bin/hermit` script.
I already removed it in one of my draft commits that I have not published yet. Did it cause you any issues locally?